sabnzbd/nzbqueue.py

```diff
@@ -441,6 +441,7 @@ def remove(self, nzo_id, add_to_history=True, save=True, cleanup=True, keep_basi
             nzo = self.__nzo_table.pop(nzo_id)
             nzo.deleted = True
             if cleanup and nzo.status not in (Status.COMPLETED, Status.FAILED):
+                # TODO: Use nzo.is_gone() ?
```

Comment:
@Safihre pretty sure we should just use is_gone here, added note for ya to review later on..